package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "fta-fmea-ui",
+  "name": "@kbss-cvut/fta-fmea-ui",
```

```suggestion
  "name": "fta-fmea-ui",
```
Not sure why adding @kbss-cvut here -- I believe "fta-fmea-ui" is sufficient as soon as it is globally unique on github. There is usecase of using @kbss-cvut if there is a collision with a same package in npmjs.org. But this is not the case.